src/__tests__/lib/utils/SynapseClient.test.tsx

```diff
@@ -5,68 +5,48 @@ import {
   FileHandleAssociateType,
   PaginatedResults,
 } from '../../../lib/utils/synapseTypes/'
-import { SynapseClient, SynapseConstants } from '../../../lib/utils/'
+import { SynapseClient } from '../../../lib/utils/'
 import { FunctionReturningPaginatedResults } from '../../../lib/utils/SynapseClient'
 
 describe('it works at integration level testing', () => {
-  it('invalid call', () => {
-    return SynapseClient.doGet(
-      '/repo/v1/invalid',
-      undefined,
-      undefined,
-      BackendDestinationEnum.REPO_ENDPOINT,
-    ).catch((error) => {
-      expect(error.status).toEqual(404)
-      expect(error.reason).toContain(
-        'GET was not found.',
+  it('Call to invalid rest endpoint results in error', async () => {
+    try {
+      await SynapseClient.doGet(
+        'repo/v1/invalid',
+        undefined,
+        undefined,
+        BackendDestinationEnum.REPO_ENDPOINT,
       )
-    })
+    } catch (error) {
+      expect(error.status).toEqual(404)
+      expect(error.reason).toContain('GET was not found.')
+    }
   })
 
   it('version call', () => {
-    return SynapseClient.getVersion()
-      .then((data) => {
-        expect(data.version).toBeDefined()
-      })
-      .catch((err) => {
-        fail(err.reason)
-      })
-  })
-
-  it('delete entity', () => {
-    return SynapseClient.deleteEntity(
-      'invalid_session_token',
-      'invalid_entity_id',
-    )
-      .then((data) => {
-        fail(
-          'should not be able to delete an entity with an invalid access token',
-        )
-      })
-      .catch((resp) => {
-        // invalid access token
-        expect(resp['status']).toBe(401)
-      })
+    return SynapseClient.getVersion().then(data => {
+      expect(data.version).toBeDefined()
+    })
   })
```

Comment:
e.g. this is really just testing that we get a 401 when Synapse could easily change to return a 404 and still be valid behavior.